doc/why_mf_next.md

```diff
@@ -0,0 +1,113 @@
+# Why `MessageFormat` needs a successor ([issue #49](https://github.com/unicode-org/message-format-wg/issues/49))
+
+## Intro
+
+The `MessageFormat` API and syntax have been around for a long time.
+
+Its “ancestor”, [java/text/MessageFormat](https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html), was introduced with Java 1.4, February 2002.
+
+The [ICU MessageFormat](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/MessageFormat.html) is tagged as stable API since ICU 3.0 (June 2004)
+
+The ICU version evolved compared to the JDK one:
+* added support for plurals (ICU 3.8, 2007)
+* added support for select (ICU 4.4, 2010)
+* named arguments (`...{user}...` vs `...{0}...`)
+* better handling of the apostrophe escaping
+* date/time/number skeletons (2018)
+* more
+
+Despite being around for such a long time, it is still not well supported by localization tools.
+
+Other efforts: Fluent, FB
```

Comment:
Does FB refer to https://github.com/facebookincubator/fbt/?
```suggestion
Other efforts: Fluent, FBT
```

Reply:
Would it be better to include links to each project?

Reply:
> Would it be better to include links to each project?

Will do.